hadoop-hdds/rocks-native/src/main/native/cplusplus_to_java_convert.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
Probably need to add a another Apache 2.0 license header before this original one?